src/com/itschool/session4/homework/binaryarithmeticoperators/ex1.java

```diff
@@ -0,0 +1,38 @@
+package com.itschool.session4.homework.binaryarithmeticoperators;
+
+import java.util.Scanner;
+
+/*Write a Java program to take two numbers as input and display the result of addition,
+subtraction, multiplication, division, and modulus.
+ */
+
+
+public class ex1 {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter the first number: ");
+        double nr1;
+        nr1 = scanner.nextDouble();

```

Comment:
since `nr1` is not being computed in some decision construct/loop, there is no need to declare it and initialize it on different lines. So instead of these two lines you should have directly:
`double nr1 = scanner.nextDouble();`
Same for `nr2`.